register.js

```diff
@@ -1,24 +1,22 @@
 import { Reaction } from "/lib/api";
 
-if (process.env.NODE_ENV === "development") {
-  // Register package as ReactionCommerce package
-  Reaction.registerPackage({
-    label: "Reaction Devtools",
-    name: "reaction-devtools",
-    icon: "fa fa-code",
-    meta: {
-      version: "0.0.1"
-    },
-    autoEnable: true,
-    registry: [
-      {
-        name: "reaction-devtools-admin",
-        label: "Dev Tools",
-        provides: ["settings"],
-        container: "dashboard",
-        template: "DevTools",
-        icon: "fa fa-code"
-      }
-    ]
-  });
-}
+// Register package as ReactionCommerce package
+Reaction.registerPackage({
+  label: "Reaction Devtools",
+  name: "reaction-devtools",
+  icon: "fa fa-code",
+  meta: {
```

Comment:
I know you copied this from the original but plugins have a top-level attribute called `version` now. Can we set this and make it `1.0.0` for this post 2.0 version?

Reply:
Sure, I added it as a top-level attribute and also bumped the meta-data one. Is that what you were expecting?